src/Reducers/Web3.js

```diff
@@ -1,17 +1,17 @@
-import * as Web3 from '../Actions/Web3'
+import * as Web3Action from '../Actions/Web3'
 
 const initialState = {
-  provider: null
```

Comment:
Does this change break the redux developer tools within Ganache? e.g., 

https://github.com/trufflesuite/ganache-cli/issues/300

In that issue, I fixed that specific bug for that person, but the issue still exists. For instance, here's the workaround necessary for The Pouncing Dead: 

https://github.com/tcoulter/cryptozombies/blob/master/resources/jsx/Actions/helpers/ReduxWeb3Provider.js#L13

Reply:
Based on your comment in the pouncing dead code, we had broken the dev tools before this change was made. I'm no longer storing the `provider` as state, I store a web3 instance now.